feature/user/src/main/java/com/exalt/feature/user/enums/GenderConfig.kt

```diff
@@ -0,0 +1,10 @@
+package com.exalt.feature.user.enums
+
+import androidx.compose.ui.graphics.Color
+import com.exalt.feature.user.R
+
+enum class GenderConfig(val icon: Int, val color: Color) {
```

Comment:
```suggestion
enum class GenderConfig(@DrawableRes val icon: Int, val color: Color) {
```
ça t'assure à la compilation que l'Int passé en paramètre fait bien référence à un drawable et pas une Color ou autre

Reply:
Oui, tout à fait! Je l'ai ajouté sur quelques modifications mais il s'est peut-être perdu après